kasa/smart/smartbulb.py

```diff
@@ -46,8 +46,9 @@ def valid_temperature_range(self) -> ColorTempRange:
         if not self.is_variable_color_temp:
             raise KasaException("Color temperature not supported")
 
-        ct_range = self._info.get("color_temp_range", [0, 0])
-        return ColorTempRange(min=ct_range[0], max=ct_range[1])
+        return cast(
+            ColorTemperatureModule, self.modules["ColorTemperatureModule"]
+        ).valid_temperature_range
```

Comment:
As this is part of the public API for now, we should have a test for this, and I thought we had?

Reply:
We didn't, only for IOT devices.  Have added this